Stackless/module/clinic/taskletobject.c.h

```diff
@@ -0,0 +1,49 @@
+/*[clinic input]
```

Comment:
argumentclinic is one thing that has completely passed me by.  I assume this is all generated code.

Reply:
Yes, the .c.h files are generated. Argument clinic supports stackless calls and works just fine.